### DISCUSSION TITLE
Icons to Headings, Paragraphs

### DISCUSSION THREAD

**Discussion by zachberry** [ucfopen/Obojobo#1841] [Ideas]
A rough mockup of what this might look like:

![image](https://user-images.githubusercontent.com/73479/119872157-290c4d80-bef1-11eb-8c83-c0a80637fafa.png)

It would be nice to be able to add an icon to a Heading or Paragraph (perhaps any piece of content). The layout of this could then be handled by Obojobo and could do various things depending on if we're at a mobile vs desktop layout

**Comment by paradisojr** [comment]
This ‘thumbs up’ made me think of how cool it would be to have some sort of emoticons (emotional state) associated with the content (at a granular level). It could provide some qualitative data around something that a lot of instructors and researchers actually care to know as well. Even a simple upvote / downvote is a low-level (subreddit) style of getting a feel for student perception of content.
